bin/assemble.js

```diff
@@ -87,7 +87,6 @@ function generateComponents() {
     :fill="displayColor"
     :transform="displayMirrored"
     v-bind="$attrs"
-    v-on="$listeners"
```

Comment:
Example the first... I've been told that in Vue you don't pass callbacks down, but it seems like that just means you need to manually make components emit every type of event anyone might want to hook into. This was my attempt at a clean solution. Is there something more idiomatic?

Reply:
For more complex components Vue gently recommends explicitly defining and distributing all component props and then turning on strict prop checking for the component, so anything extraneous errors(warns?) at runtime. This sort of use-case where the component is a thin wrapper around another element is one of the example exceptions though, and you’re not breaking any Vue conventions by doing it.

In Vue 3 the way listeners are handled has been streamlined, so all the native listeners are just on $attrs unless you explicitly emit a custom event with the same name.

This actually _worked fine_ in Vue 3 unchanged but throws a deprecation warning.

Reply:
Nice to hear that's been changed.

Reply:
I was pleasantly surprised that the fix for the deprecation notice was just to delete the line and everything kept working as expected!